<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1379</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="graph">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>591</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string>Graph</string>
    </property>
    <widget class="QCustomPlot" name="customPlot1" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>571</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_plot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>PLOT</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="serial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>SERIAL PORT</string>
    </property>
    <widget class="QComboBox" name="comboBox_baud">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_com">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>COM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Baudrate:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_connect">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>38</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_disconect">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>88</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_refresh">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tunning">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>461</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="title">
     <string>TUNNING</string>
    </property>
    <widget class="QGroupBox" name="pitch">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>PTICH</string>
     </property>
     <widget class="QLineEdit" name="p_kp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="p_ki">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="p_kd">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ki</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kd</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="servo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>SERVO</string>
     </property>
     <widget class="QLineEdit" name="s_off_r">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="s_off_l">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>S_OFFSET_R</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>S_OFFSET_L</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="motor">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>MOTOR</string>
     </property>
     <widget class="QLineEdit" name="m_off_r">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_17">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>M_OFFSET_R</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_off_l">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>M_OFFSET_L</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_send">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>330</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="roll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>ROLL</string>
     </property>
     <widget class="QLineEdit" name="r_kp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="r_ki">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="r_kd">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ki</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kd</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="yaz">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>YAZ</string>
     </property>
     <widget class="QLineEdit" name="y_kp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="y_ki">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="y_kd">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ki</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kd</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="transmitter">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>TRANSMITTER</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="receiver">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>251</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>RECEIVER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>GET-DATA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1379</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
